source/jormungandr/jormungandr/scenarios/tests/new_default_tests.py

```diff
@@ -509,3 +515,30 @@ def crowfly_in_ridesharing_test():
     assert journey.durations.car == 0
     assert journey.distances.ridesharing == 43
     assert journey.distances.car == 0
+
+
+def filter_too_much_connections_test():
+    # yes, it's a hack...
+    instance = lambda: None
+
+    instance.max_additional_connections = 10
+    mocked_pb_response = build_mocked_response()
+    mocked_request = {'debug': True, 'datetime': 1444903200, 'clockwise': True}
+    journey_filter.apply_final_journey_filters([mocked_pb_response], instance, mocked_request)
+    assert all('deleted_because_too_much_connections' not in j.tags for j in mocked_pb_response.journeys)
+
+    instance.max_additional_connections = 0
+    mocked_pb_response = build_mocked_response()
+    mocked_request = {'debug': True, 'datetime': 1444903200, 'clockwise': True}
+    journey_filter.apply_final_journey_filters([mocked_pb_response], instance, mocked_request)
+    assert sum('deleted_because_too_much_connections' not in j.tags for j in mocked_pb_response.journeys) == 8
+    # the best journey must be kept
+    assert mocked_pb_response.journeys[14].tags == [u'bike']
```

Comment:
clearer to test (same below):
```suggestion
    assert u'deleted_because_too_much_connections' not in mocked_pb_response.journeys[14].tags
```